server/controllers/async-controllers.js

```diff
@@ -86,7 +86,7 @@ const SeriesTransporter = ({series}: any) => {
           </div>
         </div>
       </Layout12>
-      <CardGrid items={series.items} />
+      <CardGrid items={series.items} hidePromoText={true} />
```

Comment:
This is a bit 🗑 - but I'm sure we'l iron it out when we look at promos across the site.